schema.json

```diff
@@ -0,0 +1,263 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"default": {
+		"edges": [],
+		"nodes": []
+	},
+	"properties": {
+		"required": true,
+		"edges": {
+			"items": {
+				"$ref": "#/definitions/JSONCanvasEdge"
+			},
+			"type": "array"
+		},
+		"nodes": {
+			"items": {
+				"$ref": "#/definitions/JSONCanvasNode"
+			},
+			"type": "array"
+		}
+	},
+	"definitions": {
```

Comment:
This must be `$defs`.

https://json-schema.org/draft/2020-12/json-schema-core#section-8.2.4